src/storhaug

```diff
@@ -358,13 +364,23 @@ setup_create_resources()
             options="_netdev,defaults,direct-io-mode=enable,transport=tcp,xlator-option=*client*.ping-timeout=10" \
         --clone ganesha_state-clone ganesha_state meta interleave="true" clone-max="${STORAGE_NUM_SERVERS}"
 
+    mkdir -p "${HA_MNT_DIR}/${HA_SMB_MNT_DIR}"
+    pcs -f ${cibfile} resource create ctdb_lock ocf:heartbeat:Filesystem \
+        params \
+            device="localhost:/${HA_SMB_VOL}" \
+            directory="${HA_MNT_DIR}/${HA_SMB_MNT_DIR}" \
+            fstype="glusterfs" \
+            options="_netdev,defaults,direct-io-mode=enable,transport=tcp,xlator-option=*client*.ping-timeout=10" \
+        --clone ctdb_lock-clone ctdb_lock meta interleave="true" clone-max="${STORAGE_NUM_SERVERS}"
+
     pcs -f ${cibfile} constraint location ganesha_state-clone rule resource-discovery=exclusive score=0 role eq storage
 
     pcs cluster cib-push ${cibfile} || storlog "ERR" "Failed to create filesystem resources."
 
     # CTDB
     pcs -f ${cibfile} resource create ctdb ocf:heartbeat:CTDB \
         params \
+            ctdb_recovery_lock="${HA_MNT_DIR}/${HA_SMB_MNT_DIR}/ctdb.lock" \
```

Comment:
@jarrpa but, ctdb_recovery_lock seems to be a required parameter of the CTDB RA?
I am using resource-agents-3.9.5-82.el7_3.4.x86_64.